WikiCohereEnglishEmbeddingOnlyStreamingScenario.cs

```diff
@@ -1,9 +1,23 @@
-﻿using Microsoft.Extensions.Configuration;
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
 
 namespace VectorIndexScenarioSuite
 {
-    internal class WikiCohereEnglishEmbeddingOnlyStreamingScenario : WikiCohereEmbeddingOnlyBaseSceario
+    internal class WikiCohereEnglishEmbeddingOnlyStreamingScenario : EmbeddingOnlyScearioBase
     {
+        protected override string BaseDataFile => "wikipedia_base";

```

Comment:
lets not duplicate these

Reply:
the streaming inherit the wikiEmbeddingOnly, eventually this can all goes to the config file